packages/block-serialization-spec-parser/test/__snapshots__/index.js.snap

```diff
@@ -10,3 +10,14 @@ Array [
   },
 ]
 `;
+
+exports[`block-serialization-spec-parser-php basic parsing parse() works properly 1`] = `
+Array [
+  Object {
+    "attrs": Array [],
```

Comment:
This is concerning that it produces different type for `attrs` property.

Should we add test which validates that both outputs generated by parsers are deeply equal?

Reply:
I'm just leaving a note that it doesn't work as expected. The second parser doesn't have this issue  :)